packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -344,7 +344,7 @@ export const validateCart = async (
   // to see this new cart state instead of what's stored on the user's browser.
   const isStale = isOrderFormStale(orderForm)
 
-  if (isStale && orderNumber) {
+  if (isStale) {
     const newOrderForm = await setOrderFormEtag(orderForm, commerce).then(
       joinItems
     )
```

Comment:
An alternative way to keep the orderNumber undefined as it was before would be to only return if `isStale && orderNumber` as it was previously, but setting the etag when `isStale` (like you did). Does that make sense?

Reply:
Change done!